### DISCUSSION TITLE
Feature Request: Configuation .ini File Backup Functionality

### DISCUSSION THREAD

**Discussion by luxmanifestus** [X2CommunityCore/xcom2-launcher#310] [Ideas]
This is a big ask but it could be useful to develop a way to backup all the customized configuration .ini files in the mod folders. In the event of a drive failure or automatic deletion (as is the case when game files are verified through Steam) or overwriting (when mods update), it would be great to have a way to load back in those configuration settings. I already keep backups of the configuration files but manually overwriting the new ones with the backups just takes too much time. If an automated process existed, it would be awesome.

**Comment by RevZero** [comment]
This has been on the "nice to have"-list for some time #53. But as you have already noticed, the implementation would take some time. Therefore, it has always been deferred in favor of other small features.
